docker/assets/prometheus/prometheus.yml

```diff
@@ -1,16 +1,10 @@
 scrape_configs:
-
-  - job_name: inx-chronicle
-    scrape_interval: 5s
-    static_configs:
-      - targets: ['inx-chronicle:9100']
-
   - job_name: hornet
     scrape_interval: 5s
     static_configs:
-      - targets: ['hornet:9311']
+      - targets: ["hornet:9311"]
```

Comment:
any particular reason for this change?

Reply:
My IDE did it 🤷

Reply:
Fine with me.

Reply:
yup, mine does the same but I have to explicitly invoke the YAML formatter.